### ISSUE TITLE
Problem with package name when gen project

### ISSUE THREAD

**Issue by duydb** [rever-tech/finatra-archetype#2]
when i run:
```
mvn archetype:generate -B -DarchetypeGroupId=scala  \
        -DarchetypeArtifactId=finatra-archetype         \
        -DarchetypeVersion=1.0 -DgroupId=company        \
        -DartifactId=projectx                           \
        -DpackageName=com.example.demo                  \
        -DarchetypeCatalog=local
```

project gen with wrong path: **com.example.demo**. I think it must be **com/example/demo**

**Comment by thanhtien522** [comment]
Belong to [ArchetypeDescriptor document](http://maven.apache.org/archetype/archetype-models/archetype-descriptor/archetype-descriptor.html) the `packageName` property should be renamed to `package`

>Filesets can be packaged, which means the selected files will be generated/copied in a directory structure that is prepended by the package property. They can be non-packaged, which means that the selected files will be generated/copied without that prepend.

**Comment by thanhtien522** [comment]
Fixed in 9a23d157605ba808babe5f99817e0233bca8f0b1
